vllm/config.py

```diff
@@ -3907,7 +3907,7 @@ class VllmConfig:
     lora_config: Optional[LoRAConfig] = None
     speculative_config: SpeculativeConfig = field(default=None,
                                                   init=True)  # type: ignore
-    decoding_config: Optional[DecodingConfig] = None
+    decoding_config: DecodingConfig = field(default_factory=DecodingConfig)
```

Comment:
why was this change required?

Reply:
mypy was raising issue about decoding_config being None, but I will remove this

Reply:
also because of L53 in `structured_output/__init__.py`